skiplang/prelude/ts/src/sk_runtime.ts

```diff
@@ -138,7 +138,7 @@ class LinksImpl implements Links {
       return this.lineBuffer.length;
     };
     this.SKIP_read_line_get = (i: int) => {
-      return this.lineBuffer[i];
+      return this.lineBuffer[i]!;
```

Comment:
Same here.

Reply:
Removed in https://github.com/SkipLabs/skip/pull/409

Reply:
Not removed, but fixed in new version.